### PR TITLE
CATTY-494 update year in license header

### PR DESCRIPTION
**Pull request by KrennKristof**
Update year in license headers

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Verify that the Jira ticket is in the status *Ready for Development*
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/Catty/Storyboard+XIB/LaunchScreen.storyboard [hunk 1]
```diff
@@ -1,11 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="11762" systemVersion="16C67" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="vwt-6C-EYF">
```
Comment (m-herold) [inline src/Catty/Storyboard+XIB/LaunchScreen.storyboard:2]: Please revert your changes to this unintentionally changes file.

**Review by m-herold — CHANGES_REQUESTED** [review]

File: src/Catty/Storyboard+XIB/LaunchScreen.storyboard (hunk 1 above)
Comment (KrennKristof) [reply src/Catty/Storyboard+XIB/LaunchScreen.storyboard:2]: reverted it.

Comment (m-herold) [reply src/Catty/Storyboard+XIB/LaunchScreen.storyboard:2]: I am afraid now the file contains even more changes. Please revise your last commit. Thanks!

Comment (KrennKristof) [reply src/Catty/Storyboard+XIB/LaunchScreen.storyboard]: now it should be resolved

**Review by m-herold — APPROVED** [review]

**Pull request merged**